react-accordion-component/my-app/src/Accordion.jsx

```diff
@@ -0,0 +1,48 @@
+import './styles.css'
+import { useState } from 'react'
+
+export default function Accordiion({data}) {
+  const [id, setId] = useState(data[0].id);
+  const [showDesc, setShowDesc] = useState(true);
```

Comment:
Using 2 state variables isn't bad, but you could, if you wanted, get by with just one: use `id === undefined` to indicate that the description should not be shown.